tests/display.rs

```diff
@@ -339,4 +339,50 @@ mod generic {
             assert_eq!(s.to_string(), "10");
         }
     }
+
+    mod bound {
+        use super::*;
+
+        #[test]
+        fn bound_simple() {
+            #[derive(Display)]
+            #[display(bound = "T1, T2")]
```

Comment:
Aren't these bounds already automatically inferred in this case?

Reply:
They are. It was kinda my point, to ensure that explicitly specifying already inferred bounds won't break stuff, but I should have made it a separate test with better name. Fixed it in latest commit.